tests/unit/test_hashtools.py

```diff
@@ -0,0 +1,161 @@
+import pytest
+from pathlib import Path
+from rxconf import RxConf, AsyncRxConf
+
+
+_RESOURCE_DIR = Path.cwd() / Path("tests/resources")
+_CONF_HASH_DIR = Path.cwd() / Path("tests/resources/conf_hashing")
+
+
+# ---------- RxConf ------------
+# def test_empty_to_equal():
```

Comment:
Let's uncomment if this is needed, else — let's remove.

Reply:
Currently these tests will not pass because of #83. These tests are important.